### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by SviatoslavVasyletskyi**

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -1,3 +1,69 @@
...
+}
+
+.nav__item:not(:last-child) {
+  margin-right: 20px;
+}
+
+.nav__link {
+  color: var(--main-color-font);
+  text-decoration: none;
+  padding: 23px 0;
```
Comment (asakevych) [inline src/style.css:45]: its better not to set paddings here, just use height: 100% or `aling-items: center` in .header

File: src/style.css [hunk 2]
```diff
@@ -1,3 +1,69 @@
...
+  --main-color-font: #000;
+  --main-color-font-is-hover: #00acdc;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+}
+
```
Comment (asakevych) [inline src/style.css:11]: 1. remember to set `box-sizing: border-box` for all elements, it's a good practice
2. I would better set font-weight  for all header elements of each link, not globally for body

**Review by asakevych — CHANGES_REQUESTED** [review]
Great job! just some small changes needed :) and please add your links here in comment - demo and tests

File: src/index.html [hunk 3]
```diff
@@ -2,13 +2,48 @@
...
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <div class="logo">
+        <a href="#home" class="logo"></a>
+        <img src="./images/logo.png" alt="logo">
+      </div>
```
[the remark this answers is not in the document]
Comment (SviatoslavVasyletskyi) [reply src/index.html:18]: done! thank for the review! actually i have a question how to add my links in comment? should i do everything the same as first time in vscode and sent it with gitbash?

File: src/style.css [hunk 4]
```diff
@@ -1,3 +1,69 @@
...
+.nav__list {
+  display: flex;
+  margin: 0;
+  padding: 0;
+  list-style-type: none;
+}
+
+.nav__item {
+  list-style: none;
+}
```
[the remark this answers is not in the document]
Comment (SviatoslavVasyletskyi) [reply src/style.css:36]: done!

File: src/style.css (hunk 2 above)
Comment (SviatoslavVasyletskyi) [reply src/style.css:11]: done!

File: src/style.css (hunk 1 above)
Comment (SviatoslavVasyletskyi) [reply src/style.css:45]: in this case i have a problem and it doesnt work when i add class .header for header and add aling-items: center or height: 100%. so i left padding without change

File: src/index.html (hunk 3 above)
[the remark this answers is not in the document]
Comment (SviatoslavVasyletskyi) [reply src/index.html]: well, i hope i got it!

**Comment by SviatoslavVasyletskyi** [comment]
links here:
demo - https://SviatoslavVasyletskyi.github.io/layout_moyo-header
test - https://SviatoslavVasyletskyi.github.io/layout_moyo-header/report/html_report

File: src/style.css [hunk 5]
```diff
@@ -1,3 +1,73 @@
...
+header {
+  display: flex;
+  padding: 0 50px;
+  justify-content: space-between;
+  align-items: center;
+  background: #fff;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+}
+
+/* .header {
```
Comment (Oskinavara) [inline src/style.css:21]: Please remove unnecessary comment

**Review by Oskinavara — CHANGES_REQUESTED** [review]
Apart from the unused comment, everything is fine now.

**Review by Oskinavara — APPROVED** [review]
Everything's fine, good job

**Review by mvjl000 — APPROVED** [review]

**Pull request closed without merging**